source/renderer/app/i18n/locales/ja-JP.json

```diff
@@ -818,6 +822,7 @@
   "wallet.transaction.type.exchange": "換金",
   "wallet.transaction.unresolvedInputAddressesAdditionalLabel": "アドレスを表示する。",
   "wallet.transaction.unresolvedInputAddressesLinkLabel": "Cardanoエクスプローラーでこのトランザクションを開き",
+  "wallet.transaction.unresolvedInputAddressesLinkUnavailableLabel": "!!!Cardano Explorer link currently unavailable.",
```

Comment:
@darko-mijic @daniloprates we need to add this translation. I have asked Junko to provide it.

Reply:
@darko-mijic @daniloprates resolved in https://github.com/input-output-hk/daedalus/pull/2034/commits/3befa418d121bab88573b74afb6b5c6a7774f741